Test/git-issues/git-issue-2747.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+function AnotherBrokenFunction(): nat {
+  var y := 0;
+  assert true by {
+    if
+    case x: bool :| true =>
```

Comment:
I would suggest changing the example so that it doesn't emit a trigger-related warning; maybe introduce a `predicate P(x: bool)` and change `true` to `P(x)`?

We could even do that for the other example as well.

Reply:
It would be nice but unfortunately, it leads to yet another (unrelated) error, and I need to do some digging to see if it has been reported yet.